theme/js/components/reuse/card.vue

```diff
@@ -6,11 +6,11 @@ Vue and simpler -->
     <Placeholder
       class="reuse-image fr-mb-3v fr-responsive-img"
       alt=""
-      :src="image_url"
+      :src="image"
       type="reuse"
     />
-    <div>
-      {{ type_label }}
+    <div class="text-grey-300">
+          {{ $tc('X datasets', datasets.length) }}
```

Comment:
Does the new API return a list of datasets or only a number ? The props says `datasets` is a Number but it's used as an Array here.